client/cypress/e2e/language.spec.js

```diff
@@ -1,11 +1,14 @@
 // / <reference types="Cypress" />
 
 describe('Translation Test', () => {
-  it('Checks that locales have correct content', () => {
+  it('Sets default language to /en and redirects', () => {
```

Comment:
Should we start using Cypress over Jest? I'm hitting odd behavior around Intl with Jest while trying to write tests.

Reply:
We should continue to use both, but for different purposes - Cypress for full end-to-end integration tests and feature tests, and jest for unit tests and snapshot tests (we've been working toward this based on internal docs, but we should document this formally in this repo!). You should file the jest issue as a bug and we can look into it